service/mantle/shopify/ShopifyServices.xml

```diff
@@ -98,4 +99,27 @@
 					 </script>
 			 </actions>
 </service>
+
+    <service verb="get" noun="LocationId">
+        <description>
+            This service gets the location Id for the specific shop
+        </description>
+        <in-parameters>
+            <parameter name="systemMessageTypeId" required="true" />
+        </in-parameters>
+        <out-parameters>
+            <parameter name="locationId" />
+        </out-parameters>
+        <actions>
+            <entity-find-one entity-name="moqui.service.message.SystemMessageRemote" value-field="systemRemoteIdentification">

```

Comment:
The name of the object on the value-field sentence could be SystemMessageRemote

Reply:
this code is outdated